netclient/functions/pull.go

```diff
@@ -90,7 +90,7 @@ func Pull(network string, iface bool) (*models.Node, error) {
 	}
 	var bkupErr = config.SaveBackup(network)
 	if bkupErr != nil {
-		logger.Log(0, "unable to update backup file")
+		logger.Log(0, "unable to update backup file for ", network)
```

Comment:
the comma in the logger.Log automatically adds a space, so unnecessary space after "for"

Reply:
fixed